.zuul.yml

```diff
@@ -0,0 +1,30 @@
+ui: mocha-bdd
+concurrency: 5
+builder: 'zuul-builder-webpack'
+browsers:
+  - name: android
+    version: latest
+  - name: chrome
+    version: latest
+    # version: [oldest, latest] # chrome@26 won't pass
+  - name: firefox
+    version: latest
+    # version: [oldest, latest] # firefox@4 won't pass
+  - name: iphone
+    version: latest  
+    # version: [oldest, latest] # Only 10.3 is passing
+
+  # Some browsers (e.g. Edge) are tring to access bundle.js.map with `//`.  This
+  # is most likely the reason why the browsers below are failing.
+
+  # - name: ie
+  #   version: latest
+  
+  # - name: safari              # safari is not passing
```

Comment:
Wow! This is concerning, no version of safari is passing? What's not passing in Safari?

Reply:
Good catch!  It's zuul/webpack setup issue.  Please see comment below.